org.spesmilo.Electrum.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+ <id>org.electrum.electrum</id>
```

Comment:
This id (`org.electrum.electrum`) doesn't match the Flatpak ID (`org.spesmilo.Electrum`).